docs/src/APIs/AbstractModels.md

```diff
@@ -6,43 +6,10 @@ CurrentModule = ClimaLand
 
 ```@docs
 ClimaLand.AbstractModel
-ClimaLand.AbstractImExModel
 ClimaLand.AbstractExpModel
-ClimaLand.make_exp_tendency
-ClimaLand.make_imp_tendency
-ClimaLand.make_compute_exp_tendency
-ClimaLand.make_compute_imp_tendency
-ClimaLand.make_update_aux
-ClimaLand.make_update_boundary_fluxes
-ClimaLand.make_set_initial_cache
-ClimaLand.make_update_cache
-ClimaLand.make_update_drivers
-ClimaLand.prognostic_vars
-ClimaLand.prognostic_types
-ClimaLand.prognostic_domain_names
-ClimaLand.auxiliary_vars
-ClimaLand.auxiliary_types
-ClimaLand.auxiliary_domain_names
-ClimaLand.initialize_prognostic
-ClimaLand.initialize_auxiliary
-ClimaLand.initialize
+ClimaLand.AbstractImExModel
 ClimaLand.name
-ClimaLand.AbstractBC
-ClimaLand.AbstractSource
-ClimaLand.source!
-ClimaLand.AbstractBoundary
-ClimaLand.TopBoundary
-ClimaLand.BottomBoundary
-ClimaLand.boundary_flux!
-ClimaLand.diffusive_flux
-ClimaLand.boundary_vars
-ClimaLand.boundary_var_domain_names
-ClimaLand.boundary_var_types
-ClimaLand.FieldMatrixWithSolver
-ClimaLand.make_jacobian
-ClimaLand.make_compute_jacobian
-ClimaLand.set_dfluxBCdY!
-ClimaLand.get_drivers
+ClimaLand.initialize
 ClimaLand.total_energy_per_area!
 ClimaLand.total_liq_water_vol_per_area!
```

Comment:
It's probably still not the best place, but I think these two functions fit better in user_diagnostics.md